package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bap-node-microframework-ping-module",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "tsc": "tsc",
+    "build": "npm run tsc && find . -type f -depth 1 -name '*.ts' | xargs -I {} tsc {} typings/index.d.ts --declaration --module commonjs",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+      "bap-node-microframework": "git+https://github.com/bap-node-microframework/node-microframework.git",
+      "bap-node-microframework-mongoose": "git+https://github.com/bap-node-microframework/node-microframework-mongoose.git"
+  }
```

Comment:
Without extends BaseControllerMongoose:
Property 'router' does not exist in type 'typeof PingController'.
And if I add a router: any property:

        router.get(getArgs[0], function (req, res) { return descriptor.value(req, res); });
              ^

TypeError: Cannot read property 'get' of undefined
    at .../api/node_modules/bap-node-microframework-demo/node_modules/bap-node-microframework/dist/decorators/routing.decorators.js:17:15
    at __decorate (.../api/node_modules/bap-node-microframework-demo/dist/app/PingController.js:10:110)
    at .../api/node_modules/bap-node-microframework-demo/dist/app/PingController.js:33:5
    at Object.<anonymous> (.../api/node_modules/bap-node-microframework-demo/dist/app/PingController.js:44:2)
    at Module._compile (module.js:541:32)

during build of application which uses ping-module